src/main/java/net/education/kyivstar/config/CreateSchema.java

```diff
@@ -0,0 +1,71 @@
+package net.education.kyivstar.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.DATABASE_PATH;
+import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.TABLES_PATH;
+import static net.education.kyivstar.services.util.Utils.readSqlScriptFromFileStream;
+
+
+public class CreateSchema {
+    private EmbeddedMariaDbLifeCycle embeddedMariaDbLifeCycle;
+
+    public CreateSchema(EmbeddedMariaDbLifeCycle embeddedMariaDbLifeCycle) {
+        this.embeddedMariaDbLifeCycle = embeddedMariaDbLifeCycle;
+        createDataBase();
```

Comment:
Initializing the database schema and possibly tables within the constructor of the CreateSchema class can lead to unexpected behavior and violates the principle of separation of concerns. Constructors should ideally be used for object initialization and not for performing complex operations such as database setup.

Reply:
Static member 'net.education.kyivstar.config.EmbeddedMariaDbLifeCycle.conf' accessed via instance reference